hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/properties/EvmConfigProperties.java

```diff
@@ -0,0 +1,38 @@
+package com.hedera.mirror.web3.evm.properties;
+
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import javax.validation.constraints.NotBlank;
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
+
+@ConfigurationProperties(prefix = "evm")
```

Comment:
The standard mirror node properties share common prefixes
```suggestion
@ConfigurationProperties("hedera.mirror.web3.evm")
```

Reply:
Eventually, we'll need this added to `docs/configuration.md` but we can probably save that for later.